jena/src/test/scala/org/w3/banana/jena/JenaRDFLoader.scala

```diff
@@ -0,0 +1,8 @@
+package org.w3.banana.jena
+
+import org.scalatest.FunSuite
```

Comment:
For scalatest, wordspec is used, but this file should not be needed, rather add the test to [JenaTest.scala](https://github.com/w3c/banana-rdf/blob/series/0.8.x/jena/src/test/scala/org/w3/banana/jena/JenaTest.scala)